docs/nidcpower/functions.rst

```diff
@@ -704,27 +704,23 @@ nidcpower.Session methods
 
     :type signal_identifier: str
 
-.. py:method:: fetch_multiple(count, timeout=datetime.timedelta(seconds=1.0))
+.. py:method:: fetch_multiple(count=None, timeout=datetime.timedelta(seconds=1.0))
 
-    Returns an array of voltage measurements, an array of current
-    measurements, and an array of compliance measurements that were
+    Returns an array of named tuples that were
```

Comment:
Why not call out the name of the named tuple?

Reply:
Added, though I'm not sure it any clearer.